recipe/recipe.yaml

```diff
@@ -0,0 +1,264 @@
+            - if not exist %LIBRARY_PREFIX%\bin\libmamba.dll (exit 1)
+            - if not exist %LIBRARY_PREFIX%\lib\libmamba.lib (exit 1)
+
+  - package:
+      name: libmambapy
+    build:
+      script: build_mamba
+      string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      skip:
+        - python_impl == 'pypy'
+    requirements:
+      run_exports:
+        - ${{ pin_subpackage('libmambapy', upper_bound='x.x') }}
+      ignore_run_exports:
+        by_name:
+          - spdlog
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib("c") }}
+        - cmake
+        - ninja
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - pybind11
+            - pybind11-abi
+      host:
+        - python
+        - scikit-build
+        - pip
+        - setuptools <79
+        - pybind11
+        - pybind11-abi
+        - openssl
+        - yaml-cpp
+        - cpp-expected
+        - spdlog
+        - fmt >=11.1.0
+        - termcolor-cpp
+        - nlohmann_json <3.12
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+        # zstd is not a direct dependency but this specification is needed to
+        # distinguish builds of libmambapy for specific specific version of zstd.
+        - zstd
+      run:
+        - python
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+    tests:
+      - python:
+          imports:
+            - libmambapy
+            - libmambapy.bindings
+      - script:
+        - python -c "import libmambapy; assert libmambapy.__version__ == '${{ version }}'"
+
+  - package:
+      name: mamba
+    build:
+      script: build_mamba
+    requirements:
+      ignore_run_exports:
+        from_package:
+          - spdlog
+          - fmt
+          - libsolv
+          - ${{ compiler('cxx') if linux }}
+          - ${{ "python" if win }}
+          - if: unix
+            then:
+              - libcurl
+              - openssl
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib("c") }}
+        - cmake
+        - ninja
+        - if: win
+          then:
+          - vcpkg-tool
+          - python
+          - curl >=8.4.0
+          - zlib
+      host:
+        - cli11 >=2.2,<3
+        - reproc
+        - reproc-cpp
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+        - spdlog
+        # zstd is not a direct dependency but this specification is needed to
+        # distinguish build of mamba for specific specific version of zstd.
+        - zstd
+      run:
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+    tests:
+      - script:
+        - mamba --help
+        - if: unix
+          then:
+            - test -f "${PREFIX}/bin/mamba"
+            - test -f "${PREFIX}/condabin/mamba"
+            # TODO: remove `etc/profile.d/mamba.sh` after 30th Sept. 2025
```

Comment:
Unrelated: This is not the case anymore given https://github.com/mamba-org/mamba/pull/3946.
```suggestion
```